### DISCUSSION TITLE
Sideband and RNSD within a LAN

### DISCUSSION THREAD

**Discussion by buhalo1** [markqvist/Reticulum#271] [General]
What interfaces are better to use to connect the Sideband that runs on the phone and the RNSD that runs on the RPi within a LAN?
"Connect over local WiFi/Ethernet" seems to be optimal, but there is no such mode in RNSD.
UDP also seems to be suitable for LAN communications but is not supported by Sideband.

**Comment by markqvist** [comment]
The "Connect over local WiFi/Ethernet" corresponds to the AutoInterface in `rnsd`. It is the only interface enabled by default, so it should work out of the box, as long as multicast is supported by your LAN hardware.

**Reply by faragher** [reply]
Does Autointerface currently run on Windows?

**Reply by buhalo1** [reply]
I get an error when starting nomadnet.

[Warning] AutoInterface[Default Interface] could not autoconfigure. This interface currently provides no connectivity.

Config.

[[Default Interface]]
    type = AutoInterface
    interface_enabled = true
    group_id = reticulum
    devices = wlan0,eth1

Apparently the error appears due to the lack of a network interface with ipv6 support. Do I understand correctly that autointerface does not support ipv4?

**Reply by Swissbandit** [reply]
i think the group id is the hindering thing.

# You can create multiple isolated Reticulum
  # networks on the same physical LAN by
  # specifying different Group IDs.

  group_id = reticulum

normally this is not needed and the interface works with every device connectable out of the box.(but i dont know about windoze)

**Reply by buhalo1** [reply]
Initially, I tried without this parameter, the result was the same.

**Reply by Swissbandit** [reply]
so if your phone with sideband is on the wifi of the pi with the node running and you still cant connect or send announces, it may be a bug in sideband or its config,only alternative would be using the phones internet and connect via testnet.
be sure to restart sb / nomad after a config change so it will be used.

udp may not be an option in sb but it may help when active on the node and the phone is on the wifi of the node.

**Reply by Swissbandit** [reply]
I remember it was working with an early sb version but may due to a bug, it was not working in more recent releases.
now i have 2 networks i dont want to connect and mix up,so i leave autointerface deactivated on my 2 nodes and dont link up my phone or tab directly. so i did not test this recently again.
